### PR TITLE
DiseasesExpanded - Cures QoL fixes

### PR DESCRIPTION
**Pull request by realwiseserpent**
Hi! It's just a bunch of small fixes for cures. Most of them are sort order change. 
Hungerms vaccine removed from Consumables tab.
Stress relief upgrade for nanobots now doesnt appear if nanobots are turned off.
I also added radiation flask to apothercary recipes, idk, it looks like you can get one from germcatcher.

### CODE REVIEW

File: DiseasesExpanded/Cures/HungermsVaccineConfig.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 namespace DiseasesExpanded
 {
-    class HungermsVaccineConfig : IEntityConfig
+    class HungermsVaccineConfig// : IEntityConfig
```
Comment (pether-pg) [inline DiseasesExpanded/Cures/HungermsVaccineConfig.cs:7]: restore IEntityConfig

File: DiseasesExpanded/Cures/MutatingAntiviralConfig.cs [hunk 2]
```diff
@@ -40,22 +40,22 @@ public GameObject CreatePrefab()
...
             MedicineInfo medInfo = new MedicineInfo(ID, EffectID, MedicineInfo.MedicineType.CureSpecific, null, new string[] { MutatingSickness.ID });
 
-            GameObject looseEntity = EntityTemplates.CreateLooseEntity(ID, 
-                STRINGS.CURES.MUTATINGANTIVIRAL.NAME, 
-                STRINGS.CURES.MUTATINGANTIVIRAL.NAME, 
+            GameObject looseEntity = EntityTemplates.CreateLooseEntity(ID,
+                STRINGS.CURES.MUTATINGANTIVIRAL.NAME,
+                STRINGS.CURES.MUTATINGANTIVIRAL.DESC,
                 1f,
-                false,
```
Comment (pether-pg) [inline DiseasesExpanded/Cures/MutatingAntiviralConfig.cs:52]: why changed to true?

File: DiseasesExpanded/Cures/RadShotConfig.cs [hunk 3]
```diff
@@ -39,11 +39,12 @@ private void DefineRecipe()
             };
             recipe = new ComplexRecipe(ComplexRecipeManager.MakeRecipeID(ApothecaryConfig.ID, (IList<ComplexRecipe.RecipeElement>)ingredients, (IList<ComplexRecipe.RecipeElement>)results), ingredients, results)
             {
-                time = 100f,
+                time = 200f,
```
Comment (pether-pg) [inline DiseasesExpanded/Cures/RadShotConfig.cs:42]: why the time is doubled?

File: DiseasesExpanded/Cures/HungermsVaccineConfig.cs (hunk 1 above)
Comment (realwiseserpent) [reply DiseasesExpanded/Cures/HungermsVaccineConfig.cs:7]: Why? The entity isn't used but appeares here:
image

File: DiseasesExpanded/Cures/AlienSicknessCureConfig.cs [hunk 4]
```diff
@@ -56,11 +56,12 @@ private void DefineRecipe()
             };
             recipe = new ComplexRecipe(ComplexRecipeManager.MakeRecipeID(ApothecaryConfig.ID, (IList<ComplexRecipe.RecipeElement>)ingredients, (IList<ComplexRecipe.RecipeElement>)results), ingredients, results)
             {
-                time = 100f,
+                time = 200f,
```
[the remark this answers is not in the document]
Comment (realwiseserpent) [reply DiseasesExpanded/Cures/AlienSicknessCureConfig.cs:59]: The cure is considered advanced so I took the same time from Serum Vial (AdvancedCureConfig).

File: DiseasesExpanded/Cures/RadShotConfig.cs (hunk 3 above)
Comment (realwiseserpent) [reply DiseasesExpanded/Cures/RadShotConfig.cs:42]: The cure is considered advanced so I took the same time from Serum Vial (AdvancedCureConfig).

File: DiseasesExpanded/Cures/MutatingAntiviralConfig.cs (hunk 2 above)
Comment (realwiseserpent) [reply DiseasesExpanded/Cures/MutatingAntiviralConfig.cs:52]: Some of the mod pills are measured in units, some - in gramms. In-game pills are all measured in units.

File: DiseasesExpanded/Cures/HungermsVaccineConfig.cs (hunk 1 above)
Comment (pether-pg) [reply DiseasesExpanded/Cures/HungermsVaccineConfig.cs:7]: each entity config must use IEntityConfig. please restore it and never remove again

Comment (realwiseserpent) [reply DiseasesExpanded/Cures/HungermsVaccineConfig.cs:7]: ok, but what about commenting out the whole class like u did with obsolete rooms in Rooms Expanded mod?

Comment (realwiseserpent) [reply DiseasesExpanded/Cures/HungermsVaccineConfig.cs:7]: I see. Hungerms Vaccine is integrated in other mods like PrintingPodRecharge. Why is the recipe commented out?

**Review by pether-pg — APPROVED** [review]

**Pull request merged**